src/main/java/org/broadinstitute/hellbender/tools/funcotator/OutputRenderer.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import htsjdk.variant.variantcontext.VariantContext;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.List;
+
+/**
+ * An abstract class to allow for writing output for the Funcotator.
```

Comment:
"writing output" is a bit vague.  Can you be a little more specific?  Would a subclass be a different file format?

Reply:
A different file format, or a stream, or an IP connection - whichever the user chooses to write.

I updated the comment with a link to the `VcfOutputRenderer`.